test/github-issues/7662/issue-7662.ts

```diff
@@ -0,0 +1,105 @@
+import { expect } from "chai";
+import { Connection } from "../../../src/connection/Connection";
+import {
+    createTestingConnections,
+    closeTestingConnections,
+} from "../../utils/test-utils";
+import { MemoryLogger } from "./memory-logger";
+
+describe("github issues > #7662 postgres extensions installation should be optional", () => {
+    it("should NOT install extensions if option is disabled", async () => {
+        let connection: Connection | null = null;
+        try {
+            const connections = await createTestingConnections({
+                entities: [`${__dirname}/entity/*{.js,.ts}`],
+                enabledDrivers: ["postgres"],
+                schemaCreate: false,
+                dropSchema: true,
+                createLogger: () => new MemoryLogger(true),
+                driverSpecific: {
+                    installExtensions: false,
+                },
+            });
+
+            expect(connections).to.have.length(1);
```

Comment:
This won't be true in all test environments.

This is why your tests are failing.

In the oracle tests it fails because we aren't running postgres there so it gets omitted.

Instead, return early if the connections length are 0 or something like that